src/config/config.h

```diff
@@ -167,6 +167,9 @@ struct Config {
   int json_max_nesting_depth = 1024;
   JsonStorageFormat json_storage_format = JsonStorageFormat::JSON;
 
+  // hash
```

Comment:
The variable name is self-explanatory so you can remove this comment. Otherwise, it would be best if you elaborated on the goal of this variable/option.

Reply:
Yes. I had polished the comment of here